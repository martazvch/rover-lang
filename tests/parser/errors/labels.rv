break :1 5

print oui

:1 {

}

print oui

:b 
