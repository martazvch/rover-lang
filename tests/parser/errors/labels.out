non_ident_label
non_ident_label
missing_brace_after_label
